json-edit.py

```diff
@@ -46,7 +46,7 @@ def save(self, filename=None):
         if filename is None:
             filename = self.filename
         with open(filename, 'w') as file:
-            json.dump(self.object, file, sort_keys=True, indent=4,
+            json.dump(self.object, file, sort_keys=True, indent=2,
```

Comment:
Rather than fight over the indent :) please revert to 4 for now. Another commit could include user configuration of settings like this - I would be open to suggestions which ideally would only require a small amount of code as I use the editor on small embedded device so want to keep both the LOC and dependencies to a minimum.

Reply:
Hello, I reverted to indent 4 (even if I find json tree more readable with indent 2 ;) )